acos_client/v30/slb/service_group.py

```diff
@@ -88,9 +95,9 @@ def create(self, name, protocol=TCP, lb_method=ROUND_ROBIN, **kwargs):
         self._set(name, protocol, lb_method, **kwargs)
 
     def update(self, name, protocol=None, lb_method=None, health_monitor=None,
-               **kwargs):
+               health_monitor_disable=0, **kwargs):
```

Comment:
0/1 is the value expected by the AXAPI so I kept it that way for consistency's sake.  I can use True/False here and perform the conversion before we send the message over.   Not sure why I just didn't pull it from kwargs...